lib/pf/UnifiedApi/Controller/Crud.pm

```diff
@@ -259,6 +266,7 @@ sub render_create {
     }
 
     my $id = $obj->{$self->primary_key};
+    $id =~ s#/#~#g;
     $self->res->headers->location($self->make_location_url($id));
     return $self->render(json => { id => $id , message => "'$id' created"}, status => $status);
```

Comment:
I believe that would display "'bob~bob' created" instead of "'bob/bob' created"

The location header should be adjusted but the ID in the message should be left untouched

Reply:
@julsemaan: you're right.

Reply:
This has been resolved in 90b1c7cbec5ec659075c1f03f6907328367e7540